{{ cookiecutter.project_slug }}/.github/workflows/docs.yml

```
@@ -0,0 +1,61 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches: [main]
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: pages
+  cancel-in-progress: false
+
+jobs:
+  # Single deploy job since we're just deploying
+  deploy:
```

We should also run the tests and make sure they pass before running the deployment:

```suggestion
jobs:
  test:
    uses: ./.github/workflows/ci.yml

  # Single deploy job since we're just deploying
  deploy:
    needs: test
    timeout-minutes: 15
```

This requires `test` to be run on `workflow_call`. I also noticed timeouts are missing in the CI files.

---

Even if we noticed they're broken, we won't be able to stop the docs from building and deploying. This will prevent that. But we definitely don't want to run the tests twice, so we either do this only on release, or we swap it around and put the `uses` call in the other file and call this workflow and only run it on the main branch.